tools/ci_build/github/android/default_qnn_aar_build_settings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "build_abis": [
+        "arm64-v8a"
+    ],
+    "android_min_sdk_version": 21,
+    "android_target_sdk_version": 24,
+    "build_params": [
+        "--enable_lto",
+        "--android",
+        "--parallel",
+        "--cmake_generator=Ninja",
+        "--build_java",
+        "--build_shared_lib",
+        "--use_qnn",
```

Comment:
do we want to enable other EPs relevant for Android here as well?

Reply:
I am not sure. @skottmckay ?

Reply:
If you are using this package I would have assumed you're targeting very specific hardware given the binary size cost. 

Given that I'm not sure why you'd enable a non-QNN EP.